File: CS/MVVMExpenses/Views/Category/CategoriesEditFormView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Linq;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using MVVMExpenses.Models;
using MVVMExpenses.Models.ViewModels;
using DevExpress.XtraGrid.Views.Grid;

namespace MVVMExpenses.Common.Views.Category {
    [DevExpress.Utils.MVVM.UI.ViewType("CategoryView")]
    public partial class CategoriesEditFormView : DevExpress.XtraEditors.XtraUserControl {
        public CategoriesEditFormView() {
            InitializeComponent();
            this.TypeImageComboBoxEdit.Properties.Items.AddEnum<DataModels.TransactionType>();
            if(!DesignMode) InitBindings();
        }

        void InitBindings() {
            var fluent = mvvmContext1.OfType<CategoryViewModel>();
            fluent.SetObjectDataSourceBinding(
                categoryBindingSource, x => x.Entity, x => x.Update());

            fluent.SetBinding(
                gridControl1, gc => gc.DataSource, x => x.CategoryTransactionDetails.Entities);

            ((GridView)gridControl1.MainView).OptionsBehavior.Editable = false;
            ((GridView)gridControl1.MainView).Columns["Category"].Visible = false;
        }
    }
}

File: VB/MVVMExpenses/Views/Category/CategoriesEditFormView.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Linq
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports MVVMExpenses.ViewModels


Namespace MVVMExpenses.Common.Views.Category
    <DevExpress.Utils.MVVM.UI.ViewType("CategoryView")>
    Partial Public Class CategoriesEditFormView
        Inherits DevExpress.XtraEditors.XtraUserControl

        Public Sub New()
            InitializeComponent()
            Me.TypeImageComboBoxEdit.Properties.Items.AddEnum(Of DataModels.TransactionType)()
            If Not DesignMode Then
                InitBindings()
            End If
        End Sub

        Private Sub InitBindings()
            Dim fluent = mvvmContext1.OfType(Of CategoryViewModel)()
            fluent.SetObjectDataSourceBinding(categoryBindingSource, Function(x) x.Entity, Sub(x) x.Update())
            fluent.SetBinding(GridControl1, Function(gc) gc.DataSource, Function(x) x.CategoryTransactionDetails.Entities)
            CType(GridControl1.MainView, DevExpress.XtraGrid.Views.Grid.GridView).Columns("Category").Visible = False
        End Sub
    End Class
End Namespace
